utils/airTableCalls.ts

```diff
@@ -1,27 +1,40 @@
 import Airtable from 'airtable';
 import dotenv from 'dotenv'
+import isDiscordId from './discordIdChecker'
 
 dotenv.config()
 // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-const base = new Airtable({ apiKey: 'keyuXGvCSuSPgPJUi' }).base('app5gBnmcY3h9txt0')
-export async function isContributor(discordIdOfCommandCaller: string) {
-    // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    const discordHandles: any[] = []
-    const table = base('Contributor')
+const BASE = new Airtable({ apiKey: process.env.AIRTABLE_TOKEN! }).base(process.env.AIRTABLE_BASE!)
```

Comment:
How will this work with the `!`? When I check it in the node runtime, this throws an error. 
I'd expect it to be:
```suggestion
const BASE = new Airtable({ apiKey: process.env.AIRTABLE_TOKEN }).base(process.env.AIRTABLE_BASE)
```
But maybe I'll learn something today!

Reply:
If you run this code in `ts-node` without the `!` you will see an error similar to the following
```bash
TSError: ⨯ Unable to compile TypeScript:
utils/airTableCalls.ts:7:72 - error TS2345: Argument of type 'string | undefined' is not assignable to parameter of type 'string'.
  Type 'undefined' is not assignable to type 'string'.

7 const BASE = new Airtable({ apiKey: process.env.AIRTABLE_TOKEN }).base(process.env.AIRTABLE_BASE)
```
The `!` tells TypeScript that we guarantee the value to be present and if it isn't there the code can happily crash.
See [non-null-assertion-operator](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-0.html#non-null-assertion-operator)

Reply:
Gotcha, so it's TS specific. Thanks for the info! Indeed when I use `ts-node` instead of `node` in my terminal, I can see this works.